source/_components/climate.markdown

```diff
@@ -20,6 +20,39 @@ climate:
+
+We explain each of the properties and the possible choices for them shortly:
+
+* The **action** property specifies the overall operational mode that the device is currently in. Choices are:
+ * Cooling
+ * Heating
+ * Auto (Cool or heat, depending on what's necessary to achieve the target temperature)
+ * Off
+* The **target_temperature** property selects the desired temperature for the device to achieve. The device will take measures to do so depending on what `action` is chosen. While most devices allow only to specify a `target_temperature` and will start heating / cooling when the temperature differs too far from that temperature, some devices allow finer control by specifying an  acceptable temperature corridor:
+  * The **target_temp_high** property selects an upper limit on the acceptable temperature. The device will start cooling (if `cooling` or `auto` are selected as `action`) if that temperature is exceeded.
+  * The **target_temp_low** property selects a lower limit on the acceptable temperature. The device will start heating (if `heating` or `auto` are selected as `action`) if that temperature is exceeded.
+* The **target_humidity** property selects the target humidity. If this is set and the device is able to dry / humidify the air, it will do so if the actual humidity differs from this value.
+* The **control** property specifies how the device is currently being controlled. Many devices allow to chose between an automated control (following a given preset, see below) or a manual control. *Note*: This does *not* influence whether the device is controlled by Home Assistant. The device is always controlled by Home Assistant, even if in `manual` mode. This property controls only the setting on the actual device. Choices are:
+ * Preset (device follows the device-specific preset, see below)
+ * Manual (the device itself is only controlled manually / via Home Assistant)
+* The **source** property specifies (for devices which support this), what source to use for achieving the goal given in the `action`. Choices are:
+ * Electric
+ * Gas
+ * Fan_only
+* The **preset** property controls which preset to follow, if `preset` is selected for the `control` property. If the devices is in `manual` mode, this setting has no effect. The choices for this property are not standardized but differ from device to device. Note that the presets must be provided by the device, either statically or by direct configuration of the device. Home Assistant does not yet allow you to create / edit presets. If you want to emulate a preset, an automation and a timer component might be what you want.
+* The **away** property can only be on or off and sets devices into a away-mode in which they usually conserve energy by heating / cooling less.
```

Comment:
So… would "away mode" possible be a candidate for another option in the `control` property? Then it would be `auto`, `manual` and `away`, basically making `away` a special kind of `auto`.

Reply:
I think that sounds fair, but I'm not sure if everyone agrees with it, and would prefer to let a separate service to handle that :-)

Reply:
If you have auto, manual and away then you would rather have: home, sleep, away, vacation, manual.

This sounds like an attractive setting if you could publish metrics around it to graph it properly.